server/src/core/xml_data.rs

```diff
@@ -6,24 +6,24 @@ use crate::{constants::{OYarn, SymType}, core::symbols::symbol::Symbol};
 
 
 #[derive(Debug, Clone)]
-pub enum XmlData {
-    RECORD(XmlDataRecord),
+pub enum OdooData {
+    RECORD(OdooDataRecord),
     MENUITEM(XmlDataMenuItem),
     TEMPLATE(XmlDataTemplate),
     DELETE(XmlDataDelete),
 }
 
 #[derive(Debug, Clone)]
-pub struct XmlDataRecord {
+pub struct OdooDataRecord {
     pub file_symbol: Weak<RefCell<Symbol>>,
     pub model: (OYarn, Range<usize>),
     pub xml_id: Option<OYarn>,
```

Comment:
Maybe also rename xml_id to something generic too? like `data_id` or something? since we also use it for CSV ?

Reply:
aha, okay then makes sense:)